server/context.go

```diff
@@ -351,20 +351,22 @@ func (ctx *Context) PGURL(user string) (*url.URL, error) {
 	} else {
 		options.Add("sslmode", "verify-full")
 		requiredFlags := []struct {
-			name  string
-			value *string
+			name     string
+			value    *string
+			flagName string
 		}{
-			{"sslcert", &ctx.SSLCert},
-			{"sslkey", &ctx.SSLCertKey},
-			{"sslrootcert", &ctx.SSLCA},
+			{"sslcert", &ctx.SSLCert, "cert"},
+			{"sslkey", &ctx.SSLCertKey, "key"},
+			{"sslrootcert", &ctx.SSLCA, "ca-cert"},
```

Comment:
i think this is missing the point of this map, but i'll let @nvanbenschoten comment on this.